playground/update.sh

```diff
@@ -1,24 +1,39 @@
-#! /bin/sh
+#!/bin/bash
```

Comment:
Okay.

I tend to use `/bin/bash` in scripts because that's the terminal I use every day, so I want the scripts to run in the same environment. But I agree it's likely it'll run with just `/bin/sh` as well (I'm jut not as well familiar with its limitations).